packages/storage-drivers/resolve-storage-mongo/src/index.js

```diff
@@ -33,7 +42,13 @@ function createDriver({ url, collection }) {
     }
 
     return {
-        saveEvent: event => getCollection().then(coll => coll.insert(event)),
+        saveEvent: event =>
+            getCollection().then(coll => coll.insert(event)).catch((e) => {
+                if (e.code === DUPLICATE_KEY_ERROR) {
```

Comment:
Should this change be reflected in related tests: https://github.com/reimagined/resolve/blob/ae633ee345a18cbcb2c251716ad22d8aede2cf1b/packages/storage-drivers/resolve-storage-mongo/test/index.tests.js ?

Reply:
ok